### PR TITLE
Adopt TestComplete-plugin

### PR DESCRIPTION
**Pull request by rinconedgar**
# Description
I would like to adopt the GitLab plugin: https://github.com/jenkinsci/testcomplete-plugin. 
My GitHub username is @rinconedgar

Needs approval from @Igor-Filin or we need to wait the two weeks described in the "Adopt a plugin" page.

# Submitter checklist for adding or changing permissions

### Always

- [x] Add link to plugin/component Git repository in description above

### When adding new uploaders (this includes newly created permissions files)

- [x] [Make sure to `@`mention an existing maintainer to confirm the permissions request, if applicable](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)
- [x] Use the Jenkins community (LDAP) account name in the YAML file, not the GitHub account name
- [x] Make sure to `@`mention the users being added so their GitHub account names are known if they require GitHub merge access (see below).
- [x] [All newly added users have logged in to Artifactory and Jira at least once](https://github.com/jenkins-infra/repository-permissions-updater/#requesting-permissions)

### Reviewer checklist (not for requesters!)

- [ ] Check this if newly added person also needs to be given merge permission to the GitHub repo (please @ the people/person with their GitHub username in this issue as well). If needed, it can be done using an [IRC Bot command](https://jenkins.io/projects/infrastructure/ircbot/#github-repo-management)
- [ ] Check that the `$pluginId Developers` team has `Admin` permissions while granting the access.
- [ ] In the case of plugin adoption, ensure that the Jenkins Jira default assignee is either removed or changed to the new maintainer.
- [ ] If security contacts are changed (this includes add/remove), ping the security officer (currently `@Wadeck`) in this pull request. If an email contact is changed, wait for approval from the security officer.

There are [IRC Bot commands](https://jenkins.io/projects/infrastructure/ircbot/#issue-tracker-management) for it

### CODE REVIEW

**Comment by MarkEWaite** [comment]
@Igor-Filin are you willing to approve this adoption request?

**Comment by NotMyFault** [comment]
Hey @rinconedgar,

is there anything particular you want to add to this plugin? I checked your profile out and noticed that this PR is your first contribution at all.
If you would like to contribute to Jenkins' ecosystem, I recommend to start with maintained plugins or detailed-described projects to gain great feedback and get familiar with the projects.
I feel, adopting and maintaining a plugin is a bit too much for a newcomer.

**Comment by rinconedgar** [comment]
Hi @NotMyFault and @MarkEWaite 
this plugin is being used by TestComplete and the developer @Igor-Filin is no longer with us and couldn't be reached. We need to get access to fix this security ihttps://www.jenkins.io/security/advisory/2023-01-24/#SECURITY-2741

**Comment by MarkEWaite** [comment]
> Hi @NotMyFault and @MarkEWaite this plugin is being used by TestComplete and the developer @Igor-Filin is no longer with us and couldn't be reached. We need to get access to fix this security https://www.jenkins.io/security/advisory/2023-01-24/#SECURITY-2741

Thanks very much @rinconedgar for being willing to adopt the plugin.  You're welcome to start your work to prepare a fix for the security issue https://www.jenkins.io/security/advisory/2023-01-24/#SECURITY-2741 .  You can submit it as a pull request from your fork to the jenkinsci repository.

The [plugin adoption rules](https://www.jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/#abandoned-plugins) require that we either need an explicit acknowledgement from @Igor-Filin that you are allowed to adopt the plugin or we need to wait two weeks of non-response from @Igor-Filin , then you can be assigned as a maintainer.

**Comment by rinconedgar** [comment]
i @NotMyFault and @MarkEWaite,

you will receive a request from @osamasalemsb that is trying to create a PR with the fix. We work together so he can be added as well

thanks

**Comment by NotMyFault** [comment]
Hey @rinconedgar,

if you wish to proceed with this PR, you need to log into [artifactory](https://repo.jenkins-ci.org/ui/login) and [Jira](https://issues.jenkins.io/).

Additionally, this PR needs authorization by @Igor-Filin / @Melligos because they are no longer inactive.

**Comment by MarkEWaite** [comment]
> Additionally, this PR needs authorization by @Igor-Filin / @Melligos because they are no longer inactive.

As far as I can tell, the two week timeout has elapsed after a response was requested, so we no longer require a response in order to allow the adoption based on the ["abandoned plugins"](https://www.jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/#abandoned-plugins) section of ["adopt a plugin"](https://www.jenkins.io/doc/developer/plugin-governance/adopt-a-plugin/).

**Comment by NotMyFault** [comment]
I would not consider the plugin abandoned any longer because a maintainer with write access returned, approved and merged several PRs.

**Comment by rinconedgar** [comment]
@MarkEWaite and @NotMyFault , I am logged in to both. Let me know what should I do next 
thanks!

File: permissions/plugin-TestComplete.yml [hunk 1]
```diff
@@ -4,4 +4,4 @@ github: "jenkinsci/testcomplete-plugin"
 paths:
   - "org/jenkins-ci/plugins/TestComplete"
 developers:
-  - "igor_filin"
+  - "rinconedgar"
```
Comment (NotMyFault) [inline permissions/plugin-TestComplete.yml:7]:
```suggestion
  - "igor_filin"
  - "rinconedgar"
```

**Review by NotMyFault — APPROVED** [review]
I'll go ahead and approve the PR, given the former author didn't reply upon repeated request, yet chose to merge the PRs of concern.
You are welcome to cut a release.

**Pull request merged**